.github/workflows/ng-renovate.yml

```
@@ -24,7 +24,7 @@ jobs:
       - uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4.0.2
         with:
           path: |
-            .github/ng-renovate/.yarn/cache
+            ~/.yarn/berry/cache
```

This actually should be cached as `.github/ng-renovate/node-modules`
```suggestion
            .github/ng-renovate/node-modules
```

`~/.yarn/berry/cache` is the global cache, but the local install cache is in `node-modules`